src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -52,8 +55,8 @@ public static void GetMinMaxThreadsTest()
         {
             int minw, minc;
             ThreadPool.GetMinThreads(out minw, out minc);
-            Assert.True(minw > 0);
-            Assert.True(minc > 0);
+            Assert.True(minw >= 0);
+            Assert.True(minc >= 0);
```

Comment:
For instance, you may call ThreadPool.SetMinThreads(0, 0) on desktop CLR.  GetMinThreads will return zeroes after that.  At present UapAot returns zeroes, and I don't see why zeroes should be treated as invalid.